metadata/picture/__init__.py

```diff
@@ -0,0 +1,70 @@
+
+from datetime import date
+from metadata import validator
+from PIL import Image
+import io
+import base64
+
+def build_picture(pic_encoded,link:str,nb_seeds:int,zoom:float,description:str=""):
+    """
+    This function builds the Picture metadata needed for the database.
+    
+    Parameters:
+    - pic_encoded (str): The picture in a string format.
+    - link (str): The link to the picture blob.
+    - nb_seeds (int): The number of seeds in the picture.
+    - zoom (float): The zoom level of the picture.
+    """
+    
+    user_data = validator.user_data(
+        description=description,
+        number_of_seeds=nb_seeds,
+        zoom=zoom
+        )
+    
+    meta_data = validator.metadata(upload_date=date.today())
+    
+    pic_properties = get_image_properties(pic_encoded)
+    
+    image_metadata = validator.image_data(
+        format=pic_properties[2],
+        height=pic_properties[1],
+        width=pic_properties[0],
+        resolution="",
+        source=link,
+        parent=""
+        )
+    
+    quality_check_metadata = validator.quality_check(
+        image_checksum="",
+        upload_check=True,
+        valid_data=True,
+        error_type="",
+        quality_score=0.0
+        )
+    
+    picture = validator.PPicture(user_data=user_data,
+                      metadata=meta_data,
+                      image_data=image_metadata,
+                      quality_check=quality_check_metadata)
+    try:
+        validator.PPicture(**picture.model_dump())
+    except:
```

Comment:
except without exception

PydanticValidationError represent any error that Pydantic can raise and you want to catch
```suggestion
    except (PydanticValidationError) as error:
       raise PictureCreationError(f"Error: {error.arg[0]}") from None
```

`from None` hide the error raise by Pydantic and give the user your package error only

Reply:
I thought it would of been best practice to show the Pydantic error, but from a user pov it makes sense. I've applied changes